pkg/syncer/syncer.go

```diff
@@ -67,13 +75,34 @@ func StartSyncer(upstream, downstream *rest.Config, resources sets.String, clust
 		specSyncer.Stop()
 		return nil, err
 	}
+	leaseClient, err := clientset.NewForConfig(upstream)
+	if err != nil {
+		specSyncer.Stop()
+		return nil, err
+	}
+	leaseController := lease.NewController(
+		clock.RealClock{},
+		leaseClient,
+		fmt.Sprintf("cluster-%s", cluster),
+		int32(leaseDurationSeconds),
+		// TODO: onRepeatedHeartbeatFailure ? closeAllConns ?
+		nil,
+		time.Duration(float64(time.Duration(leaseDurationSeconds))*leaseRenewIntervalFraction),
+		// TODO: namespace ?
+		v1.NamespaceNodeLease,
+		// TODO: newLeasePostProcessFunc ? set owner ?
```

Comment:
Again, probably at least log something here and we can figure out what else to add later.

Reply:
✅